### PR TITLE
Added tests

### PR DESCRIPTION
**Pull request by anshuman23**

### CODE REVIEW

File: test/tensorflex_test.exs [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+
+  test "Matrix Functionalities Test" do
+    mat = Tensorflex.create_matrix(2,3, [[2.2,1.3,44.5],[5.5,6.1,3.333]])
+    assert 5.5 = Tensorflex.matrix_pos(mat,2,1)
+    assert {2,3} = Tensorflex.size_of_matrix(mat)
+    assert [[2.2,1.3,44.5],[5.5,6.1,3.333]] = Tensorflex.matrix_to_lists(mat)
+  end
+
+  test "Tensor Functionalities Test" do
+    dims = Tensorflex.create_matrix(1,3,[[1,1,3]])
```
Comment (josevalim) [inline test/tensorflex_test.exs:13]: Prefer to pattern match here. For example: `{:ok, dims} = ...`. We rarely use `elem` in Elixir code.

File: test/tensorflex_test.exs [hunk 2]
```diff
@@ -0,0 +1,54 @@
+defmodule TensorflexTest do
+  use ExUnit.Case
+  doctest Tensorflex
+
+  test "Matrix Functionalities Test" do
```
Comment (josevalim) [inline test/tensorflex_test.exs:5]: It is generally bad practice to have such large tests. Instead you want to do this:

```elixir
describe "matrix functionalities" do
  test "creates matrixes" do
    ...
  end

  test "can get size of matrix" do
    ...
  end

  ...
end
```

File: test/tensorflex_test.exs [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+    {:ok, output_tensor} = Tensorflex.float32_tensor_alloc(out_dims)
+    assert [[56.349998474121094, 39.26000213623047], [109.69999694824219, 75.52000427246094], [163.04998779296875, 111.77999877929688]] = Tensorflex.run_session(graph, input_tensor, output_tensor, "input", "output")
+  end
+  
+end
+
+
+
+  
+
```
Comment (josevalim) [inline test/tensorflex_test.exs:54]: There are a bunch of unnecessary extra new lines here. :)

**Comment by anshuman23** [comment]
Noted @josevalim! I will be sure to fix these issues in the next PR :)

**Pull request merged**